playbook.yml

```diff
@@ -18,5 +18,8 @@
       omero_server_selfsigned_certificates: true
       omero_server_rootpassword: ChangeMe
     - role: ome.omero_web
+      omero_web_config_set:
+        omero.web.application_server.host: "0.0.0.0"
+        omero.web.debug: true
```

Comment:
If the goal is to demonstrate a minimal example of production deployment, do we really want to set `omero.web.debug` to `true` as this leads to a lot of noise?

Reply:
No, we actually do not want that, just as you say. We would rather want to have a working nginx in the deployment. Would you please suggest a minimal setup which would configure nginx please ?